src/CourseDashboard.cpp

```diff
@@ -105,9 +105,9 @@ void CourseDashboard::saveToFile()
     }
 }
 
-void CourseDashboard::loadFromFile()
+void CourseDashboard::loadFromFile(std::string fileName)
 {
-    std::fstream file_DB("baza danych.txt", file_DB.in);
+    std::ifstream file_DB(fileName);
     if(!file_DB.is_open())
     {
         throw "Problem to open userBase.txt";
```

Comment:
Issue - fixed filename in the error message - can be deleted or better string concatenation

Reply:
About function "save To File" I have now realized that checking if file is open is not necessary. Always new file will be created or replaced if there exist one with the same name. If I'm not right, please correct me.

Reply:
Old one can be openned in other program e.g. notepad++ and overwriting will be not possible, I think

Reply:
Rather not. If the file is opened in another program overwriting will still be possible. But we may not have permissions to open the file if someone has changed it.